includes/polling/os/technicolor.inc.php

```diff
@@ -0,0 +1,5 @@
+<?php
+
+use LibreNMS\RRD\RrdDefinition;
```

Comment:
You don't need this line.

Please also add a copyright info https://docs.librenms.org/#General/Contributing/#copyright

Reply:
This was add by the new-os.php script.
Fixed

Reply:
I'll fix that in the script

Reply:
In fact, it's not. We don't create that file.